src-vue/src/components/LaunchArgumentsSelector.vue

```diff
@@ -0,0 +1,147 @@
+            </el-check-tag>
+        </el-tooltip>
+
+        <!-- User-input tag -->
+        <el-input
+            v-if="inputVisible"
+            ref="InputRef"
+            class="fc-tag__input"
+            v-model="inputValue"
+            size="small"
+            @keyup.enter="handleInputConfirm"
+            @blur="handleInputConfirm"
+        />
+        <el-button v-else class="button-new-tag ml-1" size="small" @click="showInput">
+            + New launch argument
+        </el-button>
+    </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import { LaunchArgument } from '../utils/LaunchArgument';
+import { NorthstarState } from '../utils/NorthstarState';
+import {invoke} from "@tauri-apps/api";
+
+export default defineComponent({
+    name: 'LaunchArgumentsSelector',
+    computed: {
+        arguments(): LaunchArgument[] {
+            const officialArguments = [
+                new LaunchArgument("-disablelogs", "Disable logging and creation of log files"),
+                new LaunchArgument("-vanilla", "Disables Northstar loading"),
+                new LaunchArgument("-northstar", "Enables Northstar loading"),
+                new LaunchArgument("-dedicated", "Starts a dedicated server without video output"),
+                new LaunchArgument("-waitfordebugger", "Waits for debugger to connect before launching game"),
+                new LaunchArgument("-enablechathooks", "Enables the use of chathooks for use by mods"),
+                new LaunchArgument("-noplugins", "Disables the plugin system"),
+                new LaunchArgument("-novid", "Disables startup videos"),
+                new LaunchArgument("-nosound", "Disables all game sounds")
+            ];
+
+            return (this.localCustomArgs.concat(officialArguments))
+                .sort((a, b) => a.argumentName.localeCompare(b.argumentName));
+        },
+        containerClasses(): string {
+            return this.gamePathIsSelected ? 'fc-tags_container' : 'fc-tags_container disabled_container';
+        },
+        gamePathIsSelected(): boolean {
+            return this.$store.state.northstar_state !== NorthstarState.GAME_NOT_FOUND;
+        }
+    },
+    data: () => ({
+        inputValue: '',
+        inputVisible: false,
+
+        values: [] as boolean[],
+        localCustomArgs: [] as LaunchArgument[]
+    }),
+    methods: {
+        onChange(index: number) {
+            this.values[index] = !this.values[index];
+            this.saveLaunchArgumentsToFile();
+        },
+        saveLaunchArgumentsToFile() {
+            const newArgs = this.arguments
+                .filter((value: LaunchArgument, index: number) => {
+                    return this.values[index];
+                })
+                .map((value: LaunchArgument) => value.argumentName)
+
+            invoke<string[]>("set_launch_arguments", {
+                gamePath: this.$store.state.game_path, arguments: newArgs
+            });
```

Comment:
Don't forget to handle the promise rejection on error ;)

Reply:
Addressed in a17a3ec0b96f02fe243a0ba38894e770efa356f7